tests/core/test_library_files.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+PY_GETH_PATH = os.path.join(
+    os.path.dirname(os.path.abspath(__file__)), "..", "..", "geth"
+)
+DEFAULT_EXCEPTIONS = (
```

Comment:
We should probably add all the ones we've already created in here:

- `AttributeError`
- `KeyError`
- `TypeError`
- `ValueError` 
- `OSError`
- `NotImplementedError`
- `FileNotFoundError`